ui/desktop/src/components/Input.tsx

```diff
@@ -12,6 +12,12 @@ interface InputProps {
   onStop?: () => void;
 }
 
+const id = 'dynamic-textarea';
+
+export function getInputElement(): HTMLElement | null {
+  return document.getElementById(id);
```

Comment:
nit: can we use a ref instead of `getElementById`?

https://react.dev/learn/referencing-values-with-refs#refs-and-the-dom

Reply:
thanks, I was scratching my head trying to remember react 🤣